src/relayserver/StoredTransaction.ts

```diff
@@ -41,7 +42,11 @@ export function createStoredTransaction (tx: Transaction, metadata: StoredTransa
     gas: ethUtils.bufferToInt(tx.gasLimit),
     gasPrice: ethUtils.bufferToInt(tx.gasPrice),
     data: ethUtils.bufferToHex(tx.data),
+    // value: ethUtils.bufferToHex(tx.value),
     nonce: ethUtils.bufferToInt(tx.nonce),
+    // v: ethUtils.bufferToHex(tx.v),
```

Comment:
?

Reply:
Fixed. I had to create a different type of storage for penalization flow transactions anyways.